packages/google_atlas/lib/src/utils/lat_lng_bounds_utils.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:atlas/atlas.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart' as GoogleMaps;
+
+class LatLngBoundsUtils {
+  static GoogleMaps.LatLngBounds formLatLngBoundsFromMarkers(
+      Set<Marker> markers) {
+    double maxNorth = -90.0;
+    double maxEast = -180;
+    double minSouth = 90.0;
+    double minWest = 180.0;
+
+    if (markers == null || markers.isEmpty) {
+      return null;
+    }
+
+    markers.forEach((marker) {
+      if (marker.position.latitude > maxNorth) {
+        maxNorth = marker.position.latitude;
+      }
+      if (marker.position.latitude < minSouth) {
+        minSouth = marker.position.latitude;
+      }
+      if (marker.position.longitude > maxEast) {
+        maxEast = marker.position.longitude;
+      }
+      if (marker.position.longitude < minWest) {
+        minWest = marker.position.longitude;
+      }
+    });
+
+    //take the shorter half around the globe
+    if ((maxEast - minWest) > 180.0) {
+      final double temp = maxEast;
+      maxEast = minWest;
+      minWest = temp;
+    }
+
+    return GoogleMaps.LatLngBounds(
+      northeast: GoogleMaps.LatLng(maxNorth, maxEast),
+      southwest: GoogleMaps.LatLng(minSouth, minWest),
+    );
+  }
+
+  static GoogleMaps.LatLngBounds newLatLngBoundsFromPosition(
+      GoogleMaps.LatLngBounds origBounds, LatLng position) {
+    if (origBounds == null) {
+      return null;
+    } else if (position == null) {
+      return origBounds;
+    } else {
```

Comment:
can we remove the else since we're returning in the above cases?

Reply:
resolved